types/kue/index.d.ts

```diff
@@ -33,7 +33,8 @@ export declare class Queue extends events.EventEmitter {
     checkActiveJobTtl(ttlOptions: Object): void;
     watchStuckJobs(ms: number): void;
     setting(name: string, fn: Function): Queue;
-    process(type: string, n?: number | ProcessCallback, fn?: ProcessCallback): void;
+    process(type: string, fn?: ProcessCallback): void;
+    process(type: string, n: number, fn?: ProcessCallback): void;
```

Comment:
Just `n`, is there anything more descriptive?

Reply:
Well the source code uses `n` as the parameter name. It refers to the number of concurrent workers though.

What do you propose?